File: LeetCode/ContainsDuplicate/ContainsDuplicate.cs
public class Solution {
    public bool ContainsDuplicate(int[] nums) {
        var n = nums.Length;
        var newSet = new HashSet<int>();

        for(int i = 0; i<n; i++){
            if(newSet.Contains(nums[i])){
                return true;
            }
            else{
                newSet.Add(nums[i]);
            }
        }

        return false;
    }
}

File: LeetCode/ContainsDuplicate/ContainsDuplicate.ts
function containsDuplicate(nums: number[]): boolean {
  let newSet = new Set();
  for (let i = 0; i < nums.length; i++) {
    if (newSet.has(nums[i])) {
      return true;
    } else {
      newSet.add(nums[i]);
    }
  }

  return false;
}
